Cargo.toml

```diff
@@ -18,4 +18,5 @@ members = [
     "tools/kms-cli",
     "tools/nomad-cli",
     "tools/balance-exporter",
+    "gelato-relay",
```

Comment:
Yeah it can be used by other people which is why I made it separate crate. Left internal for ease of use / iteration speed. 

Probably doesn't make sense to add to ethers but we can break out into separate repo/crate and publish.